Sources/App/Controllers/FriendsController.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  FriendsController.swift
+//  FeistyServer
+//
+//  Created by Tiewhan Smith on 2020/04/06.
+//
+
+import Foundation
+import Vapor
+
+class FriendsController {
+  
+  public func getFriends() -> String {
+    
+    let directory = DirectoryConfig.detect()
+    let configDir = "Sources/App/ResourceFiles"
+    
+    do {
+      
+      let path =  URL(fileURLWithPath: directory.workDir)
+        .appendingPathComponent(configDir, isDirectory: true)
+        .appendingPathComponent("friends.json", isDirectory: false)
+      
+      let data = try Data(contentsOf: path)
+      
+      return String(decoding: data, as: UTF8.self)
+      
+    } catch { }
```

Comment:
Not sure it's a good idea to have an empty catch.
If we don't need it, rather use a guard let with try? Data and exit with return "{ \"data\":\"none\" }" instead of an unnecessary do catch

Reply:
Lemme quickly do that actually

Reply:
To be honest I didn't think you could do that. The whole "try?" thing. Very different approach to try catches